metricbeat/module/vsphere/datastore/datastore.go

```diff
@@ -60,46 +61,50 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 }
 
 func (m *MetricSet) Fetch() ([]common.MapStr, error) {
+
 	f := find.NewFinder(m.Client, true)
+	if f == nil {
+		return nil, errors.New("Finder undefined for vsphere.")
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline